change/@ni-nimble-components-83fac5bb-4bc6-4067-951d-def8c7404d0c.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
something seems wrong... why are there 100% turbosnaps... that doesn't seem right

Reply:
https://github.com/ni/nimble/actions/runs/9104155344/job/25027443487#step:16:153